src/llv8-constants.cc

```diff
@@ -574,6 +575,8 @@ void Frame::Load() {
 
 void Types::Load() {
   kFirstNonstringType = LoadConstant("FirstNonstringType");
+  kFirstJSObjectType =
+      LoadConstant("type_JSGlobalObject__JS_GLOBAL_OBJECT_TYPE");
```

Comment:
Is this a new constant? Just asking, because if it is, I'd like to add it to Node's postmortem metadata test.

Reply:
Yep

Reply:
Looks like it's already part of the test.